lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -40,6 +71,23 @@ function cmd.cursePlayers( callingPlayer, targetPlayers, effectName, durationMin
         type( effectName ) == "string" and
         effectName ~= "random" and
         ( not shouldUncurse or effectName ~= "all" )
+    local amount = isSpecificEffect and not shouldUncurse and tonumber( effectName ) or nil
+
+    if amount == 1 then
+        amount = false
+        isSpecificEffect = false
+        effectName = "random"
+    end
+
+    if amount then
+        if amount < 1 or amount > multiCurseLimit or math.floor( amount ) ~= amount then
+            ULib.tsayError( callingPlayer, "Invalid amount: " .. amount )
+
+            return
+        end
```

Comment:
Why bother people with an error at all?
```suggestion
        amount = math.floor( amount )
        amount = math.Clamp( amount, 1, multiCurseLimit )
```